sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/TransactionCoordinator.java

```diff
@@ -101,4 +107,16 @@
                 }
             });
     }
+
+    public Mono<Void> dispose() {
+        if (isDisposed.getAndSet(true)) {
+            return Mono.empty();
+        }
+
+        return Mono.fromRunnable(() -> {
+            if (sendLink != null) {
+                sendLink.dispose();
```

Comment:
How was the `sendLink` disposed before implementing the interface?

Reply:
This send link is maintained by `ReactorSession`  in  `ConcurrentMap<String, LinkSubscription<AmqpSendLink>> openSendLinks`   and in its `ReactorSession.dispose()` will clean it up.